### DISCUSSION TITLE
Uploading PDFs and Files to Courses?

### DISCUSSION THREAD

**Discussion by RachelBrodsky** [inducer/relate#866] [Q&A]
Is there a way to upload pdfs or files to Relate courses? (Perhaps files for students to download, or even just view in the platform?)

**Comment by inducer** [comment]
Sure! The sample repo shows how to do it.

- Link: https://github.com/inducer/relate-sample/blob/a4c33d801a994312b10f8e7822ebe9871fd8ade2/course.yml#L26
- Uploaded file with access permissions: https://github.com/inducer/relate-sample/tree/main/pdfs
